Mathlib/Logic/Equiv/Basic.lean

```diff
@@ -832,6 +832,15 @@ def sumPiEquivProdPi {ι ι'} (π : ι ⊕ ι' → Type*) :
   left_inv f := by ext (i | i) <;> rfl
   right_inv _ := Prod.ext rfl rfl
 
+/-- The space of functions from `PUnit` into a family of types
+is isomorphic to the value of this function at `()`.
+-/
+def pUnitPiEquiv (f : PUnit → Type*) : (∀ t : PUnit, f t) ≃ f () where
```

Comment:
Isn't this [`Equiv.piUnique`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Logic/Equiv/Defs.html#Equiv.piUnique)?

Reply:
This is fine for my application so I've switched everything over to this (which makes the name of this branch a bit weird but never mind)